libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -240,6 +240,11 @@
         LOGIN_SOCIAL_ACCOUNTS_NEED_CONNECTING,
         LOGIN_SOCIAL_ERROR_UNKNOWN_USER,
         LOGIN_WPCOM_BACKGROUND_SERVICE_UPDATE,
+        SIGNUP_BUTTON_TAPPED,
+        SIGNUP_EMAIL_BUTTON_TAPPED,
+        SIGNUP_GOOGLE_BUTTON_TAPPED,
+        SIGNUP_TERMS_OF_SERVICE_TAPPED,
+        SIGNUP_DISMISSED,
```

Comment:
👍  for adding Tracks from the get go!

Reply:
Agreed. :)
Is there an easy way to get later all the Tracks pieces added in this milestone?

Reply:
@folletto , since we're using a feature branch to gather the work on SignUp and SiteCreation, the final PR to merge to `develop` will have a quite easy way to see all the events added. We will just need to checkout the `AnalyticsTracker.java` file. Does that sound practical to you?

Reply:
I can't evaluate properly — if it's simple to do, awesome! :D

Reply:
Example (taken from when the new Login was merged into develop): https://github.com/wordpress-mobile/WordPress-Android/pull/6423/files#diff-578f7e9e6f43c37a5ca1ee823a493989